  <launch>
  <node pkg="plan_env" name="dynamic_obj_generator_node" type="dynamic_obj_generator" output="screen">

    <!-- <param name="obj_generator/startX" value="10"/>
    <param name="obj_generator/endX" value="20"/> -->

    <param name="obj_generator/obj_num" value="8"/>    
    <param name="obj_generator/x_size" value="20.0"/>  <!--!!!!!!!!!!!!!-->
    <param name="obj_generator/y_size" value="20.0"/>  
    <param name="obj_generator/h_size" value="2"/>   
    <param name="obj_generator/vel" value="1"/>
    <param name="obj_generator/vel2" value="1.1"/>
    <param name="obj_generator/yaw_dot" value="2.0"/>   
    <param name="obj_generator/acc_r1" value="1.0"/>   
    <param name="obj_generator/acc_r2" value="1.0"/>   
    <param name="obj_generator/acc_z" value="0.0"/>   
    <param name="obj_generator/scale1x" value="0.3"/>   
    <param name="obj_generator/scale2x" value="0.5"/>    
    <param name="obj_generator/scale1y" value="0.3"/>   
    <param name="obj_generator/scale2y" value="0.5"/>  
    <param name="obj_generator/scale1z" value="1"/>   
    <param name="obj_generator/scale2z" value="3"/> 
    <param name="obj_generator/interval" value="100.0"/>    
    <param name="obj_generator/input_type" value="1"/> <!-- 1: Vel input, 2: Acc input-->
    <param name="obj_generator/resolution" value="0.1"/>
  </node>

  <node pkg="rviz" name="rviz_node" type="rviz" output="screen"
    args="-d $(find plan_env)/config/rviz.rviz"/>

  </launch>